2.Objects and Functions/script.js

```diff
@@ -0,0 +1,68 @@
+// object task
+
+/*let car = {
+    color: 'black'
+}
+
+car.color = 'green';
+car.engine = function func() { console.log("engine power") };
+*/
+
+//Pears and apples
+
+/*let pears = 12;
+let apples = 22;
+function sum(x, y) {
+    return x + y;
+}
+sum(pears, apples);
+*/
+
+//payment terminal
+
+/*
+let name = prompt("Enter Name: ");
+function nameDefine(x) {
+    if (x !== "") {
+        console.log(`Hello ${x}`);
+    } else {
+        console.log("no such name");
+    }
+}
+
+nameDefine(name);
+*/
+
+//Type of argument
+
+/*function type(x) {
+    console.log(typeof (x))
+}
+*/
+
+//Prime Number
+
+/*let num = prompt("Enter number: ");
+let count = 0;
+function primeCounter(x) {
```

Comment:
In this task, you need to do, determine whether the number is prime, this function does something incomprehensible. Entering 19 gives 22